packages/cache/src/options.ts

```diff
@@ -46,6 +46,13 @@ export interface DownloadOptions {
    * @default 30000
    */
   timeoutInMs?: number
+
+  /**
+   * Time after which download should be aborted if stuck
+   *
+   * @default 3600000
+   */
+  abortTimeInMs?: number
```

Comment:
This is also a timeout. Previous timeout is a chunk timeout  hence we can name this as `segmentTimeoutInMs`?